HydroModelBuilder/GWModelBuilder.py

```diff
@@ -1378,16 +1380,16 @@ def interpolate_points2mesh(self, points_obj, values_dataframe, feature_id=None,
 
     @staticmethod
     def _findInterval(row, times):
-        """TODO: docs
+        """Find interval of the model time periods.
 
-        :param row:
-        :param times:
+        :param row: row from a pandas dataframe that has the column 'datetime'
+        :param times: times is a list of datetime objects
```

Comment:
1) I try to remember to give type information in docstrings, e.g.

:param times: list[datetime], interval datetimes in model run

2) I would change the parameter `row` to `target_datetime` (assuming that's what the row represents) because technically it's not a row array and its not a very informative name.